### PR TITLE
Setup CICD for TFRecorder Repo

### PR DESCRIPTION
**Pull request by mbernico**

### CODE REVIEW

File: .github/workflows/python-cicd.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Lint with pylint
+      run: |
+        make pylint
+
+    - name: Test with pytest
```
Comment (cfezequiel) [inline .github/workflows/python-cicd.yml:30]: Are we using pytest?

File: .github/workflows/python-cicd.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
```
Comment (cfezequiel) [inline .github/workflows/python-cicd.yml:14]: We are supporting Python 3.6+ in our PRD, so maybe we can set this on the lower bound.

File: .github/workflows/python-cicd.yml (hunk 1 above)
Comment (mbernico) [reply .github/workflows/python-cicd.yml:30]: No, this is a left over, I'll fix.

File: .github/workflows/python-cicd.yml (hunk 2 above)
Comment (mbernico) [reply .github/workflows/python-cicd.yml:14]: Can try it out and make sure it works.  AFAIK we haven't tested 3.6 so far.

**Pull request merged**